topology/yaml_topology_schema.md

```diff
@@ -10,46 +10,59 @@ format below. We use an intuitive, loose format. The multiplicity is presented a
 
     # comments
     {1} const:
-    {?} keys:                              {8}
+    {?} authentication_keys:                              {8}
         {+} - id: <24-bit key number>
         {1}   algorithm: [hmac-sha-256]
         {1}   secret: <string>   
-        {?}   private-secret: <string>     {7}
+        {?}   private-secret: <string>                    {7}
     {1} shards: 
         {+} - id: <64-bit integer shard identifier> 
             {1}   nodes: 
-                {*}   - name: <node name string>
+                {*}   -  name: <node name string>
                 {?}      passive (1)
-                {1}      level: [<numerical level> | undefined | leaf | leaf-2-leaf | top-of-fabric ] (2) 
+                {1}      level: [<numerical level> | undefined | leaf | leaf-2-leaf | top-of-fabric ] {2}
```

Comment:
Any particular reason why you changed the round brackets to curly brackets?

Reply:
nothing particular, probably mistype

Reply:
footnotes are bracket or parentheses inconsistent all over the place. I left it like that, cosmetics